api/src/main/scala/pl/touk/nussknacker/engine/api/exception/NuExceptionInfo.scala

```diff
@@ -1,5 +1,8 @@
 package pl.touk.nussknacker.engine.api.exception
 
 import pl.touk.nussknacker.engine.api.Context
+import pl.touk.nussknacker.engine.api.component.ComponentType.ComponentType
 
-case class NuExceptionInfo[T <: Throwable](nodeId: Option[String], throwable: T, context: Context) extends Serializable
\ No newline at end of file
+case class NuExceptionInfo[T <: Throwable](componentInfo: Option[ExceptionComponentInfo], throwable: T, context: Context) extends Serializable
+
+case class ExceptionComponentInfo(nodeId: String, name: String, typ: ComponentType)
```

Comment:
`name` and `typ` is related with component and `nodeId` is related with usage of this component (node). It would be nice to group it somehow. I see two options: 1) Let's wrap name and typ with some class like `ComponentIdData` (could be placed somewhere near `ComponentId` and `ComponentType` classes 2) At least prefix names of field with `component`

Reply:
Ok, looks like first option is better suited in this case.